submissions/IhorOzerov/html-css-popup/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>papap</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <nav>
+        <ul class="header-icons">
+
+            <li class="greatest">
+                <label for="sq">
+                    <img class="h-image" src="icons/popup-button.png" alt="button">
+                </label>
+                <input tabindex="1" id="sq" type="checkbox" class="visually-hidden" />
+
+                <ul class="menu">
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/apps.png" alt="apps">
+                            <span>Apps</span></a></li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/blogs.png" alt="blogs">
+                            <span>Blogs</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/calendar.png"
+                                alt="calendar"><span>Calendar</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/chat.png" alt="chat">
+                            <span>Chat</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/classroom.png"
+                                alt="classroom"><span>Classroom</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/collection.png"
+                                alt="collection"><span>Collection</span></a></li>
+                    <li><a class="menu-a" href="#"><img class="visible-img" src="icons/contacts.png"
+                                alt="contacts"><span>Contacts</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/documents.png"
+                                alt="documents"><span>Documents</span></a></li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/files.png"
+                                alt="files"><span>Files</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/mail.png"
+                                alt="mail"><span>Mail</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/maps.png"
+                                alt="maps"><span>Maps</span></a>
+                    </li>
+                    <li><a class="menu-a" href="#"> <img class="visible-img" src="icons/mars.png"
+                                alt="mars"><span>Mars</span></a>
+                    </li>
+
+                    <li class="more-btn greatest"><label class="more-btn" for="more">More</label>
```

Comment:
All text inside controls must be aligned, please fix it.
Also, when the second part of popup is opened this button must hide.
![image](https://user-images.githubusercontent.com/16196199/183469636-420c2214-ef09-4750-ab76-fcc16518d9fb.png)

Reply:
Text aligned but not hiding when the second part of popup is visible
![image](https://user-images.githubusercontent.com/16196199/183584991-cd20a944-5791-4c10-b21b-5d935f282d4d.png)